CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 ### Added
 
 - For CLI development, there is now a `--allowOlderSdkVersion` param for the `coda upload` command that builds a new version. Coda will soon default to preventing a Pack build to have an older SDK version than the prior version, under the assumption that it most often happens when multiple dev environments conflict with each other. This new option is a bypass for that protection, for the rare case when you actually want to downgrade the SDK.
+- Added a new `parameters` property to our `Sync` interface so that all sync formulas, not just metadata formulas, can read their sync table parameters.
```

Comment:
Perhaps specify that what sync formulas are not metadata formulas, to break it down all the way for makers? Is it just 2way autocomplete?

Reply:
Technically property options is a metadata formula. I don't think there are any non-metadata formulas that would be gaining new access to sync table parameters from this?...

Could simplify the changelog entry to something like "Added a new `parameters` property in the `Sync` interface with sync table parameters."